Python/README.md

```diff
@@ -10,13 +10,26 @@ Basic implementation of the Sereal protocol in Python.
 
 ### Decoder behaviour settings
 
+    object_factory: Function to customize the construction of an object.
+        The default function is:
+        def _default_object_factory(classname, data):
+            return {
+                'class': classname,
+                'object': data,
+            }
+
     bin_mode_classic: Specify the binary mode to use when decoding BINARY tags.
-        If bin_mode_classic is True the decoder will auto decode all BINARY tags as UTF-8 encoded
-        If bin_mode_classic is False the decoder will read BINARY tags verbatim
-        The default is True
+        If bin_mode_classic is True, the decoder will auto decode all BINARY tags as UTF-8 encoded.
+        If bin_mode_classic is False, the decoder will read BINARY tags verbatim.
+        The default is True.
+
+    re_bytes: Specify if the regular expressions will work with strings or with bytes.
+        If re_bytes is True, the decoder will compile the regular expression Pattern as bytes.
+        If re_bytes is False, the decoder will compile the regular expression Pattern as str.
+        The default is False.
 
 ### Miscellaneous
 
 - Python 3.5+
 - No Python encoder
-- Not Perl compatible
\ No newline at end of file
+- Not Perl compatible
```

Comment:
This is not correct. The reason I removed this is because the decoder is not compatible with a languages but instead it needs to respect the Sereal specifications and thus means it's language independent.

Reply:
Sorry, but I don't understand your comment, as this line had no changes other than whitespace (trailing `\n` at the end of the file; as it is common in unix-like systems that all lines are terminated by a newline). I can revert this whitespace change anyway.